neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
Ne ze bych moc rozumel vicenasobne dedicnosti v Pythonu, ale tohle by mohl byt problem, kdybychom chteli mit vic takovych trid, ktere by do enkoderu pridavaly nejakou funkcnost dedenim.

Nebylo by lepsi, aby `Attentive` nedelal v konstruktoru nic a inicializoval se az zavolanim nejake metody?

Jeste je moznost neresit to vubec dedicnosti.